### PR TITLE
[Term Entry] Python pillow .resize()

### PR DESCRIPTION
**Pull request by Elgittos**
### Description

- Added new term entry on the '.resize()' method for the Pillow library in Python.
- Added the pillow folder and all it's subfolders in the content folder docs/content/pillow/concepts/image/terms/resize/resize.md, aswell as the resize.md file. 

### Issue Solved

- Closes Issue #4628 

### Type of Change

- Adding a new entry

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pillow/concepts/image/terms/resize/resize.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+img_resized.show()
+
+print(img_resized.height, img_resized.width)
+```
+
+The output will show the new dimensions of the image in pixels.
+
+```shell
+1000 1000
+```
```
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/resize/resize.md:123]: Just one example is to be added, the other two can be removed.

File: content/pillow/concepts/image/terms/resize/resize.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  - Image.Resampling.BILINEAR or Image.BILINEAR
+  - Image.Resampling.HAMMING
+  - Image.Resampling.BICUBIC or Image.BICUBIC
+  - Image.Resampling.LANCZOS or Image.LANCZOS
+
+Returns an **image** **object**.
+
+## Example 1
+
+`.resize()` takes pixels as arguments directly in the first parameter.
```
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/resize/resize.md:44]:
```suggestion
The following example demonstrates the use of  the `.resize()` method to change the size of the image:
```

File: content/pillow/concepts/image/terms/resize/resize.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+img = Image.open('image_path.jpg')
+
+# Will resize the image to 1000x1000 pixels.
+img_resized = img.resized((1000, 1000))
+
+# Will open the image.
+img_resized.show()
+
+print(img_resized.height, img_resized.width)
+```
```
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/resize/resize.md:59]: For this example, make use of an image to also show the resized image in the output. Keep the Image used and the Output image under the "Media" folder in the docs.

File: content/pillow/concepts/image/terms/resize/resize.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+- resample filters available
+  - Image.Resampling.NEAREST or Image.NEAREST
+  - Image.Resampling.BOX
+  - Image.Resampling.BILINEAR or Image.BILINEAR
+  - Image.Resampling.HAMMING
+  - Image.Resampling.BICUBIC or Image.BICUBIC
+  - Image.Resampling.LANCZOS or Image.LANCZOS
+
+Returns an **image** **object**.
```
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/resize/resize.md:40]: For the parameters description, please use the following format:

- `size`: Parameter description
- `resample`: Parameter description
- `box`: Parameter description
- `reducing_gap`: Parameter description

You can avoid giving the details of the resample filters for this entry. Only the parameters description should be given.

File: content/pillow/concepts/image/terms/resize/resize.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+`.resize()` takes pixels as arguments directly in the first parameter.
+
+```py
+from PIL import Image
+
+# Original image is 2000x2000 pixels.
+img = Image.open('image_path.jpg')
+
+# Will resize the image to 1000x1000 pixels.
+img_resized = img.resized((1000, 1000))
```
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/resize/resize.md:53]:
```suggestion
img_resized = img.resize((1000, 1000))
```

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @Elgittos! Thank you for contributing to the Codecademy Docs! The entry was nicely written.

I've suggested some changes for this entry, could you please review and modify those at your earliest convenience? Feel free to discuss if you have any concerns!
Thank you!

**Comment by Elgittos** [comment]
Hi thanks. I've made the changes. Hope it helped 👍

File: content/pillow/concepts/image/terms/resize/resize.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
+---
+Title: '.resize()'
+Description: 'The .resize() method is part of the Image class in the Pillow library (PIL Fork) in Python, and it is used to change the size of an image.'
```
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/resize/resize.md:3]:
```suggestion
Description: 'Changes the size of an image to a specified size.'
```

File: content/pillow/concepts/image/terms/resize/resize.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+Subjects:
+  - 'Data Science'
+  - 'Web Design'
+  - 'Web Development'
```
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/resize/resize.md:7]:
```suggestion
  - 'Computer Science'
  - 'Data Science'
  - 'Web Design'
  - 'Web Development'
```

File: content/pillow/concepts/image/terms/resize/resize.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+| Parameter Name | Description                                               |
+| -------------- | --------------------------------------------------------- |
+| `size`         | Decides the resize dimentions.                            |
+| `resample`     | Specifies the resampling filter to be used when resizing. |
+| `box`          | A 4-tuple defining the region of the image to resize.     |
+| `reducing_gap` | Used for large scale reduction to help preserve quality.  |
+
+Returns an **image** **object**.
```
Comment (mamtawardhani) [inline content/pillow/concepts/image/terms/resize/resize.md:34]: For the parameters description, do not add a table, please use the following format:

- size: Parameter description
- resample: Parameter description
- box: Parameter description
- reducing_gap: Parameter description

**Review by mamtawardhani — CHANGES_REQUESTED** [review]
Hey @Elgittos! I guess you missed some changes requested earlier, I've mentioned those again. Also, please make sure to describe the parameters in the suggested format and not the table format. Feel free to discuss if you have any concerns.

Thank you!

**Comment by Elgittos** [comment]
Hi. I corrected the entry. I hope I didn't miss anything 😄

**Review by mamtawardhani — APPROVED** [review]
Hello @Elgittos, thanks for making the required changes!

Looks good for a second review 😄

**Review by mohitO12-dev — APPROVED** [review]
ALL SORT OF FORMAT ISSUE IS SOLVED. after discussion with @mamtawardhani

**Pull request merged**